api/src/main/java/org/apache/iceberg/actions/MigrateTable.java

```diff
@@ -50,6 +50,15 @@ default MigrateTable dropBackup() {
     throw new UnsupportedOperationException("Dropping a backup is not supported");
   }
 
+  /**
+   * @param numReaders the number of concurrent file read operations to use per partition
+   * @return this for method chaining
+   */
+  default MigrateTable withParallelReads(int numReaders) {
```

Comment:
can we simplify and unify all the naming? We can just call it `parallelism` and use this name across the board. It also matches the naming in `TableMigrationUtil`

Reply:
/LGTM。  @RussellSpitzer @aokolnychyi @szehon-ho @jackye1995 when you get a chance could you please take a look? Thanks。

Reply:
Thanks, I see you changed some places, there are 2 places I think can be also consistent, the first is this place, where we can just do `default MigrateTable withParallelism(int parallelism)` instead of using a different name and introduce a `reader` concept.

The other place is in the procedure, where you have `        ProcedureParameter.optional("max_concurrent_read_datafiles", DataTypes.IntegerType)
`, we can just name the option `parallelism` as well.

What do you think?

Reply:
Thanks for your advice,It looks good to me,  but in function `importSparkPartitions` https://github.com/kingeasternsun/iceberg/blob/feature/add-parallelism-add-files/spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java#L620-L630 already has a local variable named `parallelism`

Reply:
@jackye1995  Ok, I have fixed it.

Reply:
Thank very much for your reviews  @jackye1995 , but for some reason I am not quite able to continue updating this MR In time, So I agree with your suggestion that “open a new PR” . Best wishes.